LCM/Code/App/vesc_uasrt.c

```diff
@@ -6,18 +6,18 @@
 uint8_t VESC_RX_Buff[80];
 uint8_t VESC_RX_Flag = 0;
 
-#ifdef GTV
-#define FIRMWARE_ID "GTV_2_1_4"
-#endif
-#ifdef XRV
-#define FIRMWARE_ID "XRV_2_1_4"
-#endif
-#ifdef PINTV
-#define FIRMWARE_ID "PintV_2_1_4"
-#endif
-#ifdef ADV
-#define FIRMWARE_ID "ADV_2_1_4"
-#endif
+// #ifdef GTV
+// #define FIRMWARE_ID "GTV_2_1_4"
+// #endif
+// #ifdef XRV
+// #define FIRMWARE_ID "XRV_2_1_4"
+// #endif
+// #ifdef PINTV
+// #define FIRMWARE_ID "PintV_2_1_4"
+// #endif
+// #ifdef ADV
+// #define FIRMWARE_ID "ADV_2_1_4"
+// #endif
 
```

Comment:
Defined in `LCM_Light_Control_IO_WS2812_New.csolution.yml` by selecting a device